src/lib/utils.js

```diff
@@ -90,3 +116,30 @@ export function sortBy (propertyName, desc = false) {
 export function asyncMap (array, asyncCallback) {
   return Promise.all(array.map(asyncCallback));
 }
+
+/**
+ * Create an array of numbers, starting from `start` and ending at `end`.
+ *
+ * @param {number} start - The number to start with.
+ * @param {number} end - The number to end with.
+ * @return Array<number>
+ */
+export function range (start, end) {
+  const s = Math.min(start, end);
+  const e = Math.max(start, end);
+  const result = [];
+  for (let i = s; i <= e; i++) {
+    result.push(i);
+  }
+  return result;
+}
+
+/**
+ *
+ * @param {number} min
+ * @param {number} max
+ * @return {number}
+ */
+export function random (min, max) {
```

Comment:
**🎯 suggestion:** Maybe point out when we expect or output integers in place of numbers.